src/encoding.c

```diff
@@ -9,6 +9,7 @@
 #include "wordlist-english.h"
 #include "util.h"
 
+#include <stdint.h>
```

Comment:
You mentioned these <stdint.h> includes are no longer necessary for Arduino? If so, let's remove them.

Reply:
They *should* be there.